src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -699,3 +699,46 @@ g.test('storage_texture,usage')
       });
     }, !isValid);
   });
+
+g.test('storage_texture,mip_level_count')
+  .desc(
+    `
+    Test that the mip level count of the resource of the BindGroup entry as a descriptor is 1 if the
+    BindGroup entry defines storageTexture. If the mip level count is not 1, a validation error
+    should be generated.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('mipLevelCount', [1, 2, 3, 4])
```

Comment:
perhaps also add a case like `{ baseMipLevel: 1, mipLevelCount: 1 }`. Maybe do 2x2 or 2x3 cases, 2x4 is probably not necessary.

Reply:
ok, I update the cases like below, how about this?

      .combine('baseMipLevel', [1, 2])
      .combine('mipLevelCount', [1, 2])